sdk/core/core-http/lib/util/xml.browser.ts

```diff
@@ -94,19 +96,19 @@ function domToObject(node: Node): any {
   return result;
 }
 
-// tslint:disable-next-line:no-null-keyword
-const doc = document.implementation.createDocument(null, null, null);
 const serializer = new XMLSerializer();
 
 export function stringifyXML(obj: any, opts?: { rootName?: string }) {
   const rootName = (opts && opts.rootName) || "root";
-  const dom = buildNode(obj, rootName)[0];
+  // tslint:disable-next-line:no-null-keyword
+  const doc = document.implementation.createDocument(null, null, null);
```

Comment:
I couldn't find the discussion thread for the motivation behind using a new doc for each time `stringifyXML` is called vs the previous model where the same doc was used again and again. Did you find the need to do this as part of your work or was this part of code clean up?

Reply:
Yes this was needed and in process used a cleaner solution.

Addressed all comments so far with code changes, unless commented otherwise
@ramya-rao-a @chradek

Reply:
My concern is if this will change existing behaviors for any of the libraries that are using xml.
I am not familiar with how the Document object works and the effects of using the same object vs a new one each time. We can take this offline

Reply:
As discussed offline, please look into reverting this change.

Reply:
Ran browser tests on local and was able to verify that the original changes work fine - reverted as discussed.

Looks like createDocument() gives an object that doesn't depend on any document and is intended to expose the DOM element management utilities - will carry out some more testing to rule out few other edge cases.

Reply:
Took a look at the documentation - it's confirmed that createDocument() doesn't depend on any document since its created using the DOM implementation

"The DOMImplementation interface represents an object providing methods which are not dependent on any particular document. Such an object is returned by the Document.implementation property."

https://developer.mozilla.org/en-US/docs/Web/API/DOMImplementation
Recall discussing with @chradek about this as well.